package.json

```diff
@@ -20,3 +20,5 @@
   },
   "homepage": "https://github.com/niksudan/butt#readme"
 }
+
+//I am coder now
```

Comment:
Come on dude, putting a space after // is all the range these days

Reply:
Whitespace slows down applications tremendously, I'd like to advice removing it all from the code.

Reply:
I'm sorry Dave, I can't do that.